src/osgParticle/Particle.cpp

```diff
@@ -101,7 +101,9 @@ bool osgParticle::Particle::update(double dt, bool onlyTimeStamp)
     if (onlyTimeStamp) return true;
 
     //Compute the current texture tile based on our normalized age
-    int currentTile = _start_tile + static_cast<int>(x * getNumTiles());
+    // BEGIN VRV_PATCH
```

Comment:
It is unclear if all the particle changes made it in or if they are even needed anymore. I think we are good if the old OSG particle systems look ok in VRV and not worry about the performance improvements since we have our own particle system that is separate from OSG now.